app/Dockerfile.prod

```diff
@@ -76,6 +80,7 @@ RUN chown -R app:app $APP_HOME
 COPY --from=builder --chown=app:app $APP_HOME/.venv $APP_HOME/.venv
 COPY --from=builder --chown=app:app $APP_HOME/eventyay/static/global-nav-menu $APP_HOME/eventyay/static/global-nav-menu
 COPY --from=builder --chown=app:app $APP_HOME/eventyay/static/schedule-editor $APP_HOME/eventyay/static/schedule-editor
+COPY --from=builder --chown=app:app $APP_HOME/eventyay/webapp/dist $APP_HOME/eventyay/webapp/dist
```

Comment:
One more thing - that will not work in production. 
All files that need to be served via nginx need to be in the static directory. This one is shared with the nginx component.

Please follow the example above and use
```
RUN OUT_DIR=/home/app/web/eventyay/static npm run build --prefix=eventyay/webapp
...
COPY --from=builder --chown=app:app $APP_HOME/eventyay/static/webapp $APP_HOME/eventyay/static/webapp
```